modules/eco-node-health-check-operator-creating-node-health-check.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+//nodes/nodes/eco-node-health-check-operator.adoc
+
+:_content-type: PROCEDURE
+[id="eco-node-health-check-operator-creating-node-health-check_{context}"]
+= Creating a node health check
+Using the web console, you can create a node health check to identify unhealthy nodes and specify the remediation type and strategy to fix them.
+
+.Procedure
+
+. From the *Administrator* perspective of the {product-title} web console, click *Compute* -> *NodeHealthChecks* -> *CreateNodeHealthCheck*.
+. Specify whether to configure the node health check using the *Form view*  or the *YAML view*.
+. Enter a *Name* for the node health check. The name must consist of lower case, alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character.
+. Specify the *Remediator* type, and *Self node remediation* or *Other*. The Self node remediation option is part of the Self Node Remediation operator that is installed with the Node Health Check operator. Selecting *Other* requires an *API version*, *Kind*, *Name*, and *Namespace* to be entered, which then points to the remediation template resource of a remediator.
+. Make a *Nodes* selection by specifying the labels of the nodes you want to remediate. The selection matches labels that you want to check. If more than one label is specified, the nodes must contain each label. The default value is empty, which selects both worker and master nodes.
++
+[NOTE]
+====
+When creating a node health check with the Self Node Remediation operator, you must select either `node-role.kubernetes.io/worker` or `node-role.kubernetes.io/master` as the value.
+====
++
+. Specify the minimum number of healthy nodes (in percentage or number) required for a *NodeHealthCheck* to remediate nodes in the targeted pool. If the number of healthy nodes equals to or exceeds the limit set by *Min healthy*, remediation occurs. The default value is 51%.
```

Comment:
ISG nit: avoid parentheses in general text if possible. So a possible change could look something like the following:
```suggestion
. Specify the minimum number of healthy nodes, using either a percentage or a number, required for a *NodeHealthCheck* to remediate nodes in the targeted pool. If the number of healthy nodes equals to or exceeds the limit set by *Min healthy*, remediation occurs. The default value is 51%.
```

Reply:
> ISG nit: avoid parentheses in general text if possible. So a possible change could look something like the following:

Thanks for your feedback Jesse. I have applied the update as you suggest.